src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -249,35 +283,75 @@ public void ldpcvDoesNotSupportPost(final String uri) {
+            final Response postResponse = doPostUnverified(timeMapURI);
+            //verify correct error range
+            postResponse.then().statusCode(clientErrorRange());
+            //verify presence of constraints link
+            confirmPresenceOfConstrainedByLink(postResponse);
+        } else {
+            //if it supports post but not with the Memento-Datetime header
+            if (!hasHeaderValueInMultiValueHeader("Vary-Post", MEMENTO_DATETIME_HEADER, getResponse)) {
+                final Response postResponse = doPostUnverified(timeMapURI, new Headers(
+                    new Header(MEMENTO_DATETIME_HEADER, "Sat, 1 Jan 2000 00:00:00 GMT")));
+                //verify correct error range
+                postResponse.then().statusCode(clientErrorRange());
+                //verify presence of constraints link
+                confirmPresenceOfConstrainedByLink(postResponse);
+            }
+        }
     }
 
     /**
      * 4.3.4
      *
      * @param uri The repository root URI
      */
-    //@Test(groups = {"MAY"})
+    @Test(groups = {"MAY"})
     @Parameters({"param1"})
     public void ldpcvMayDisallowPut(final String uri) {
         final TestInfo info = setupTest("4.3.4", "ldpcvMayDisallowPut",
                                         "Implementations MAY disallow PUT.",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-put",
                                         ps);
 
+        final Response createResponse = createVersionedResource(uri, info);
+        final String originalResource = getLocation(createResponse);
+        final Response originalResponse = doGet(originalResource);
+        final String timeMapURI = getTimeMapUri(originalResponse).toString();
+        final Response getResponse = doGet(timeMapURI);
+        if (hasHeaderValueInMultiValueHeader("Allow", "PUT", getResponse)) {
+            fail("This Fedora implementation allows PUTs on LDPCv.");
+        }
+
     }
 
     /**
      * 4.3.5
      *
      * @param uri The repository root URI
      */
-    //@Test(groups = {"MAY"})
+    @Test(groups = {"MAY"})
     @Parameters({"param1"})
     public void ldpcvMayDisallowPatch(final String uri) {
         final TestInfo info = setupTest("4.3.5", "ldpcvMayDisallowPatch",
                                         "Implementations MAY disallow PATCH",
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-patch",
                                         ps);
+        final Response createResponse = createVersionedResource(uri, info);
+        final String originalResource = getLocation(createResponse);
+        final Response originalResponse = doGet(originalResource);
+        final String timeMapURI = getTimeMapUri(originalResponse).toString();
+        final Response getResponse = doGet(timeMapURI);
+        if (hasHeaderValueInMultiValueHeader("Allow", "PATCH", getResponse)) {
```

Comment:
It is also conceivable that `PATCH` is allowed without be represented in an Allow header.
I would suggest also trying to perform a `PATCH`.